pkg/cli/info/info.go

```diff
@@ -0,0 +1,129 @@
+/*
+ Copyright (C) 2018 OpenControl Contributors. See LICENSE.md for license.
+*/
+
+package info
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/opencontrol/compliance-masonry/pkg/cli/clierrors"
+	"github.com/opencontrol/compliance-masonry/pkg/lib"
+	"github.com/opencontrol/compliance-masonry/pkg/lib/common"
+	"github.com/opencontrol/compliance-masonry/tools/certifications"
+	"github.com/opencontrol/compliance-masonry/tools/constants"
+	"github.com/spf13/cobra"
+	"github.com/tg/gosortmap"
+)
+
+// NewCmdInfo allows you to query for implementation_status.
+func NewCmdInfo(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "info",
+		Short: "Compliance search",
```

Comment:
@timothy-spencer let's make this `Get Compliance information from documentation` or something similar.